### PR TITLE
Allow user to mark the task as complete

### PR DESCRIPTION
**Pull request by SukiTsang**

### CODE REVIEW

File: src/main/java/seedu/task/storage/XmlAdaptedTask.java [hunk 1]
```diff
@@ -65,6 +66,7 @@ public Task toModelType() throws IllegalValueException {
 //        final DateTime closeTime = new DateTime(this.closeTime);
 //        final boolean isImportant = this.isImportant;
         final UniqueTagList tags = new UniqueTagList(taskTags);
-        return new Task(name, tags); //(name, openTime, closeTime, isImportant, tags)
+        final boolean isComplete=false;
```
Comment (Skaty) [inline src/main/java/seedu/task/storage/XmlAdaptedTask.java]: Why is this false? Why not load from the XML?

File: src/main/java/seedu/task/model/task/Task.java [hunk 2]
```diff
@@ -15,29 +15,30 @@
...
         // open time, urgent, and close time can be null
         assert !CollectionUtil.isAnyNull(name, tags);
         this.name = name;
         //TODO: set default values
         //this.openTime = openTime;
         //this.closeTime = closeTime;
         //this.isImportant = isImportant;
-        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.tags = new UniqueTagList(tags);
+        this.isCompleted=false;  // protect internal tags from changes in the arg list
```
Comment (Skaty) [inline src/main/java/seedu/task/model/task/Task.java]: `isCompleted` is passed into the constructor. Why false?

Also, that comment is for the previous line.

**Review by Skaty — CHANGES_REQUESTED** [review]
Please check and rectify. Also, please ensure that your program conforms to Java coding standards.

File: src/main/java/seedu/task/storage/XmlAdaptedTask.java (hunk 1 above)
Comment (SukiTsang) [reply src/main/java/seedu/task/storage/XmlAdaptedTask.java]: Fixed

**Review by Skaty — APPROVED** [review]

**Pull request merged**